### DISCUSSION TITLE
Can the Create New Tab button of the vertical tab bar be moved below the last tab?

### DISCUSSION THREAD

**Discussion by JamYiz** [Floorp-Projects/Floorp#559] [Feature Request]
like Edge
![图片](https://github.com/Floorp-Projects/Floorp/assets/44351690/5b403e2b-de66-48ab-93a2-7723adc5b850)
If it is placed at the bottom, it will be very troublesome to operate the mouse.
![图片](https://github.com/Floorp-Projects/Floorp/assets/44351690/2894e6ea-4215-405b-a4cd-f3c85e7fb65e)

**Comment by jezewskiBNetrox** [comment]
@JamYiz YES! I agree. This is so annoying. Can devs look into this if it is possible to implement easily? I ask for option, not default change.

**Reply by JamYiz** [reply]
Now I use Tree style Tab instead of the default vertical tab bar,it's much easier to use.

**Comment by Ryosuke-Asano** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/6290d8fd-9dbb-4829-96e4-9e6d031f856f)

Setting added.
